documentation/docs/01-project-structure.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Project Structure
+---
+
+Typicall structure of SvelteKit project is something like:
+
+```sh
```

Comment:
Not sure what use so comments are shown as comments but everything else as standard text.

Reply:
What about using the skeleton app structure, and speaking about option folders like `params` or generated folders like `.svelte-kit/` and `build/` when speaking about them in their parents' descriptions?

This is the output I've got using tree (`tree -aC --dirsfirst --gitignore` added `/` manually):

```sh
├── src/
│   ├── routes/
│   │   └── index.svelte
│   ├── app.d.ts
│   └── app.html
├── static/
│   └── favicon.png
├── tests/
│   └── test.js
├── .eslintrc.cjs
├── .gitignore
├── .npmrc
├── .prettierrc
├── README.md
├── jsconfig.json
├── package.json
├── playwright.config.js
└── svelte.config.js
```

Reply:
I think it's better if all of important files are included, so it's really fast visible to where put for example `params/` folder, that doesn't exist yet without need to search it deeper in docs.

Reply:
I agree, that's a very important folder of which, BTW, I didn't know about. ☺️ 
But I would keep it super simple to avoid to overwhelm the reader with "advanced" information, still mentioning them in their related section in the documentation page.
In this case, the section where we explain what the `src/` folder is about. 
Makes sense?

Reply:
Thinking about it, that's true. For sure, I've already learned something new looking at it. ☺️ 👍